project/test/resources/OPConsultantRecord-Output.json

```diff
@@ -1,2688 +1,2143 @@
+      },
+      "subject": {
+        "reference": "urn:uuid:9886e72f-be96-48f1-8c12-c98dc1d79033",
+        "display": "Patient"
+      },
+      "effectiveDateTime": "2024-07-01T16:29:32+05:30",
+      "performer": [ {
+        "reference": "urn:uuid:4440d5b0-47c9-4ec0-96aa-2cbd1a33be47",
+        "display": "Practitioner"
+      } ],
+      "valueString": "Regular"
+    }
+  }, {
+    "fullUrl": "urn:uuid:b83dcc39-8919-44cc-8b0d-d331fa7f6407",
+    "resource": {
+      "resourceType": "ServiceRequest",
+      "id": "b83dcc39-8919-44cc-8b0d-d331fa7f6407",
+      "status": "active",
+      "intent": "order",
+      "code": {
+        "text": "Chest CT"
+      },
+      "subject": {
+        "reference": "urn:uuid:9886e72f-be96-48f1-8c12-c98dc1d79033",
+        "display": "Patient"
+      },
+      "occurrenceDateTime": "2013-05-08T09:33:27+07:00",
+      "requester": {
+        "reference": "urn:uuid:Practitioner/example1"
+      }
+    }
+  }, {
+    "fullUrl": "urn:uuid:f76d14bf-a0dc-4538-907e-79997dc58f75",
+    "resource": {
+      "resourceType": "ServiceRequest",
+      "id": "f76d14bf-a0dc-4538-907e-79997dc58f75",
+      "status": "active",
+      "intent": "order",
+      "code": {
+        "text": "Thyroxine (T4) free [Mass/​volume] in Serum or Plasma"
+      },
+      "subject": {
+        "reference": "urn:uuid:9886e72f-be96-48f1-8c12-c98dc1d79033",
+        "display": "Patient"
+      },
+      "occurrenceDateTime": "2015-08-27T09:33:27+07:00",
+      "requester": {
+        "reference": "urn:uuid:Practitioner/example2"
+      }
+    }
+  }, {
+    "fullUrl": "urn:uuid:546e73cf-9641-4e05-8c3e-af57ac090a4c",
+    "resource": {
+      "resourceType": "MedicationStatement",
+      "id": "546e73cf-9641-4e05-8c3e-af57ac090a4c",
+      "meta": {
+        "versionId": "0",
+        "lastUpdated": "2024-07-01T16:29:32.638+05:30",
+        "profile": [ "https://nrces.in/ndhm/fhir/r4/StructureDefinition/MedicationStatement" ]
+      },
+      "status": "completed",
+      "medicationCodeableConcept": {
+        "coding": [ {
+          "system": "http://snomed.info/sct",
+          "code": "721912009",
+          "display": "Medication summary document"
+        } ],
+        "text": "Ongoing Drugs"
+      },
+      "subject": {
+        "reference": "urn:uuid:9886e72f-be96-48f1-8c12-c98dc1d79033",
+        "display": "Patient"
+      },
+      "effectivePeriod": {
+        "start": "2014-02-01T05:30:00+05:30"
+      },
+      "dateAsserted": "2014-02-22T05:30:00+05:30",
+      "derivedFrom": [ {
+        "reference": "Patient/pat1"
+      } ],
+      "reasonCode": [ {
+        "coding": [ {
+          "system": "http://snomed.info/sct",
+          "code": "323732002",
+          "display": "Amoxicillin (as amoxicillin trihydrate) 25 mg/mL oral suspension"
+        } ],
+        "text": "Amoxicillin"
+      } ],
+      "note": [ {
+        "text": "Father indicates they miss the occasional dose"
+      } ],
+      "dosage": [ {
```

Comment:
I would skip dosage for medicationStatement.  For medicationRequest dosage is ok. MedicationStatement are entered typically what medication is patient currently or in the past taken (eg patient came for some other ailment but is taking diabetes meds). In such case - dosage.text is enough, imo